### DISCUSSION TITLE
PerfectPrime TC0304, 4-Ch

### DISCUSSION THREAD

**Discussion by Kaz-Arai** [artisan-roaster-scope/artisan#1644] [Q&A]
I appreciate it if someone would tell me about the workability of the "PerfectPrime TC0304, 4-Ch" on the Artisan".

**Comment by poundy** [comment]
supported devices are listed here: https://artisan-scope.org/devices/index
Specifically, the meters supported are listed https://artisan-scope.org/devices/meters/
The 4channel version is not listed. There's a chance it uses different communication options than the 2, and there's nothing that says the 4 channel would be supported. 
According to the protocol document that is on their website https://perfectprime.com/products/tc0304 (which it would have been easy for you to link to in your original question to ensure there was no confusion the device you were referring to) it appears it's a 9600 baud connection, which appears on the surface to match the other 4-channel devices that also go from a 19200baud on the 2-ch to 9600 on the 4ch, so there's a chance this is also a common meter that will "just work" using the Center 304 setup - but likely only testing will tell.

**Reply by Kaz-Arai** [reply]
Thank you very much for your comment. I've already consulted the model-match explanation you mentioned, but as you know, it seems to work, but there was no direct description of the PerfetcPrime TC0304, and didn't have the confidence to purchase it ... Anyway I will challenge to purchase it and will share the result in this channel later.

**Reply by poundy** [reply]
someone has to "risk" the first time purchase. The good news though is that, since they have a wonderful protocol description document, should it not work it would be less work to implement than if you had to guess the protocol details...
